README.md

```diff
@@ -115,13 +115,17 @@ Configuration for log digests per email:
     # (optional, default: WARNING).
     ckanext.maillog.digest.log_level = DEBUG
 
-    # Full path of the digest log 
-    # (optional, default: /srv/app/log/maillog/debug.log).
-    ckanext.maillog.digest.log_path = /srv/app/log/ckanext-myplugin/debug.log
+    # Name of the digest log 
+    # (optional, default: debug.log).
+    ckanext.maillog.digest.log_name = my-plugin_warning.log
+    
+    # Path of the digest log 
+    # (optional, default: /srv/app/log/maillog/).
+    ckanext.maillog.digest.log_path = /srv/app/log/ckanext-myplugin/
 
     # Max bytes of digest-log-file to be kept 
     # (optional, default: 5 MB).
-    ckanext.maillog.digest.log_path = /srv/app/log/ckanext-myplugin/debug.log
+    ckanext.maillog.digest.log_path = /srv/app/log/ckanext-myplugin/
```

Comment:
```suggestion
```

Duplicated line that is stuck in between the config option `ckanext.maillog.digest.max_bytes` and the comment for it.